src/client/content_cao.cpp

```diff
@@ -1865,5 +1884,33 @@ std::string GenericCAO::debugInfoText()
 	return os.str();
 }
 
+void GenericCAO::updateMeshCulling()
+{
+	if (!m_is_local_player)
+		return;
+
+	// Grab the active player scene node so we know there's
+	// at least a mesh to occlude from the camera.
+	irr::scene::ISceneNode *node = getSceneNode();
+	if (!node)
+		return;
+
+	if (m_client->getCamera()->getCameraMode() == CAMERA_MODE_FIRST) {
+		// Hide the mesh by culling faces with normals facing
+		// the camera. Serious hackyness but it works for our
```

Comment:
```suggestion
		// Hide the mesh by culling both front and
		// back faces. Serious hackyness but it works for our
```

Reply:
Consider annotating this as `// HACK Hide the mesh by culling faces with normals facing the camera`